README.md

```diff
@@ -1,7 +1,7 @@
 Project Ramda
 =============
 
-A practical functional library for Javascript programmers.
+Ramda is the Lego® of functional libraries for Javascript.
```

Comment:
Lego is silly, ok. But the current statement would fit LoDash or Underscore just as well wouldnt it ?
How about
`A simple & practical functional library for Javascript programmers.`